presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcMetadataConfig.java

```diff
@@ -45,13 +47,17 @@ public void testExplicitPropertyMappings()
                 .put("metadata-cache-ttl", "1h")
                 .put("metadata-cache-refresh-interval", "10s")
                 .put("metadata-cache-maximum-size", "100")
+                .put("metadata-cache-thread-pool-size", "50")
+                .put("metadata-jdbc-cache-enabled", "true")
```

Comment:
You might want to test another case where this property is not in the property and False should be expected.

Reply:
that is done in "testDefaults" test above

Reply:
Do we also need to test the branch where "session.getProperty(JDBC_METADATA_CACHE_ENABLED, Boolean.class)" throws an exception? Is there a way to generate the coverage report like JaCoCo?